### ISSUE TITLE
manual/javaGuide/main/http/JavaResponse.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#954]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/http/JavaResponse.md
- diffs between [2.2.0/manual/javaGuide/main/http/JavaResponse.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/javaGuide/main/http/JavaResponse.md) and [2.3.5/manual/javaGuide/main/http/JavaResponse.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/http/JavaResponse.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/main/http/JavaResponse.md 2013-07-29 23:52:24.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/javaGuide/main/http/JavaResponse.md    2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Manipulating the response

 ## Changing the default Content-Type
@@ -6,45 +7,25 @@

 For example:

-'''java
-Result textResult = ok("Hello World!");
-'''
+@[text-content-type](code/javaguide/http/JavaResponse.java)

 Will automatically set the `Content-Type` header to `text/plain`, while:

-'''java
-Result jsonResult = ok(jerksonObject);
-'''
+@[json-content-type](code/javaguide/http/JavaResponse.java)

 will set the `Content-Type` header to `application/json`.

-This is pretty useful, but sometimes you want to change it. Just use the `as(newContentType)` method on a result to create a new similiar result with a different `Content-Type` header:
+This is pretty useful, but sometimes you want to change it. Just use the `as(newContentType)` method on a result to create a new similar result with a different `Content-Type` header:

-'''java
-Result htmlResult = ok("<h1>Hello World!</h1>").as("text/html");
-'''
-
-You can also set the content type on the HTTP response:
-
-'''java
-public static Result index() {
-  response().setContentType("text/html");
-  return ok("<h1>Hello World!</h1>");
-}
-'''
+@[custom-content-type](code/javaguide/http/JavaResponse.java)

-## Setting HTTP response headers
+You can also set the content type on the HTTP response context:
+
+@[context-content-type](code/javaguide/http/JavaResponse.java)

-You can add (or update) any HTTP response header:
+## Setting HTTP response headers

-'''java
-public static Result index() {
-  response().setContentType("text/html");
-  response().setHeader(CACHE_CONTROL, "max-age=3600");
-  response().setHeader(ETAG, "xxx");
-  return ok("<h1>Hello World!</h1>");
-}
-'''
+@[response-headers](code/javaguide/http/JavaResponse.java)

 Note that setting an HTTP header will automatically discard any previous value.

@@ -54,15 +35,17 @@

 You can easily add a Cookie to the HTTP response:

-'''java
-response().setCookie("theme", "blue");
-'''
-
-Also, to discard a Cookie previously stored on the Web browser:
-
-'''java
-response().discardCookies("theme");
-'''
+@[set-cookie](code/javaguide/http/JavaResponse.java)
+
+If you need to set more details, including the path, domain, expiry, whether it's secure, and whether the HTTP only flag should be set, you can do this with the overloaded methods:
+
+@[detailed-set-cookie](code/javaguide/http/JavaResponse.java)
+
+To discard a Cookie previously stored on the web browser:
+
+@[discard-cookie](code/javaguide/http/JavaResponse.java)
+
+Make sure, if you set a path or domain when setting the cookie, that you set the same path or domain when discarding the cookie, as the browser will only discard it if the name, path and domain matches.

 ## Specifying the character encoding for text results

@@ -72,11 +55,6 @@

 The encoding can be specified when you are generating the `Result` value:

-'''java
-public static Result index() {
-  response().setContentType("text/html; charset=iso-8859-1");
-  return ok("<h1>Hello World!</h1>", "iso-8859-1");
-}
-'''
+@[charset](code/javaguide/http/JavaResponse.java)

 > **Next:** [[Session and Flash scopes | JavaSessionFlash]]
\ No newline at end of file

```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1124
